vitest.config.ts

```diff
@@ -19,6 +19,7 @@ export default defineConfig({
     hookTimeout: TEST_TIMEOUT + 1000,
     testTimeout: TEST_TIMEOUT + 1000,
     exclude: [...defaultExclude, '**/fixtures/**'],
+    restoreMocks: true,
```

Comment:
This just seemed like a sane default, I don't know why it's not enabled by default in vitest.